fl4health/strategies/basic_fedavg.py

```diff
@@ -59,51 +51,44 @@ def __init__(
         weighted_aggregation: bool = True,
         weighted_eval_losses: bool = True,
     ) -> None:
-        """Federated Averaging strategy.
-
-        Implementation based on https://arxiv.org/abs/1602.05629
-
-        Parameters
-        ----------
-        fraction_fit : float, optional
-            Fraction of clients used during training. In case `min_fit_clients`
-            is larger than `fraction_fit * available_clients`, `min_fit_clients`
-            will still be sampled. Defaults to 1.0.
-        fraction_evaluate : float, optional
-            Fraction of clients used during validation. In case `min_evaluate_clients`
-            is larger than `fraction_evaluate * available_clients`, `min_evaluate_clients`
-            will still be sampled. Defaults to 1.0.
-        min_fit_clients : int, optional
-            Minimum number of clients used during training. Defaults to 2.
-        min_evaluate_clients : int, optional
-            Minimum number of clients used during validation. Defaults to 2.
-        min_available_clients : int, optional
-            Minimum number of total clients in the system. Defaults to 2.
-        evaluate_fn : Optional[
-            Callable[
-                [int, NDArrays, Dict[str, Scalar]],
-                Optional[Tuple[float, Dict[str, Scalar]]]
-            ]
-        ]
-            Optional function used for validation. Defaults to None.
-        on_fit_config_fn : Callable[[int], Dict[str, Scalar]], optional
-            Function used to configure training. Defaults to None.
-        on_evaluate_config_fn : Callable[[int], Dict[str, Scalar]], optional
-            Function used to configure validation. Defaults to None.
-        accept_failures : bool, optional
-            Whether or not accept rounds containing failures. Defaults to True.
-        initial_parameters : Parameters, optional
-            Initial global model parameters.
-        fit_metrics_aggregation_fn: Optional[MetricsAggregationFn]
-            Metrics aggregation function, optional.
-        evaluate_metrics_aggregation_fn: Optional[MetricsAggregationFn]
-            Metrics aggregation function, optional.
-        weighted_aggregation: bool, Optional.
-            Defaults to True, determines whether parameter aggregation is a linearly weighted average or a uniform
-            average. FedAvg default is weighted average by client dataset counts.
-        weighted_eval_losses: bool, Optional
-            Defaults to True, determines whether losses during evaluation are linearly weighted averages or a uniform
-            average. FedAvg default is weighted average of the losses by client dataset counts.
+        """
+        Federated Averaging with Flexible Sampling. This implementation extends that of Flower in two ways. The first
```

Comment:
Wanted to double check we are intentionally removing the ArXiv reference (line 64 above).

Reply:
Nope. I've added it back. Thanks for the catch.